coursework2/src/main/java/oursework2/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,37 @@
+package oursework2.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import oursework2.model.Question;
+import oursework2.service.JavaQuestionService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+    private final JavaQuestionService javaQuestionService;
+
+    public JavaQuestionController(JavaQuestionService javaQuestionService) {
```

Comment:
```suggestion
    public JavaQuestionController(QuestionService javaQuestionService) {
```
по интерфейсу надо делать инъекцию, а не реализации (см. полиморфизм, L в SOLID)